### ISSUE TITLE
Mac: CMake is mixing different versions of python libraries in pynabo.so

### ISSUE THREAD

**Issue by pomerlef** [ethz-asl/libnabo#49]
**Problem**: this [CMakeList.txt](https://github.com/ethz-asl/libnabo/blob/master/python/CMakeLists.txt) is mixing different versions of python in it's configuration. This leads to the following:

```
$ python -c "import pynabo; help(pynabo.NearestNeighbourSearch)"
Segmentation fault: 11
```

This happen when multiple version of python are installed. For me, boost +python were installed with MacPorts but similar problem are to be expected with Homebrew: https://github.com/Homebrew/homebrew/issues/25118

**The temporary fix**: use `ccmake` in your build folder and change manually the variables related to python. In my case, for python 2.7 installed from MacPorts:

```
 PYTHON_EXECUTABLE = /opt/local/bin/python                                                                                                                               
 PYTHON_INCLUDE_DIR = /opt/local/Library/Frameworks/Python.framework/Versions/2.7/Headers                                                                                 
 PYTHON_LIBRARY =  /opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/libpython2.7.dylib 
 PY_NUMPY = /opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/numpy 
```

**Comment by davidlandry93** [comment]
Once the pull request #57 is merged this should not be a problem as python 3 will be supported.

**Comment by HannesSommer** [comment]
Assuming this to be solved since the merge of PR #57 . Please reopen if not.
